### ISSUE TITLE
Full screen mode

### ISSUE THREAD

**Issue by toxinu** [jigish/slate#90]
It could be nice to have an option to disable slate in full screen mode.

What do you think about it ?

**Comment by jigish** [comment]
Hmm, what exactly do you mean by disable? Do you mean preventing bindings from firing? That does make sense because if you're in fullscreen you don't really have any need to do anything that Slate currently offers. I'll look into it.

**Comment by toxinu** [comment]
Yes, disable, cause if you use full screen you don't need slate in this mode.
Not very important but, simple option to prevent any windows movement by slate could be nice.
